### ISSUE TITLE
[BUG] \set{} command disables all inline math mode previews

### ISSUE THREAD

**Issue by ghost** [artisticat1/obsidian-latex-suite#452]
# Description
Math mode preview disables when using \set{} command in inline math. Display mode appears to work fine.

# To Reproduce
Use \set{} in inline math.

# Expected Behavior
To continue displaying math mode preview after using the \set{} command

# Additional Note
This is a minor issue as previews can be turned back on by restarting the plug-in

**Comment by superle3** [comment]
It seems to be a bug on either Obsidian's side or Mathjax side? I am not sure which one but calling `window.Mathjax.tex2chtml("\\set{")` causes an error that causes the whole preview extension to break, but `\set{}` is fine though. 

I would advise to either use `\{ \}` or create a snippet that converts `\set` -> `\set{}`.
For example the snippet:
```js
const snippet = {trigger: "\\set", replacement: "\\set{$0}", options: "m"}
```
where you use tab/space to activate it.

As to a long term solution, the author is inactive so it will take some time for a solution to be merged and I don't know an elegant way to resolve this.

In

https://github.com/artisticat1/obsidian-latex-suite/blob/ce31511a47949e3d4d0b3a43444949fd5a6a69f6/src/editor_extensions/math_tooltip.ts#L53-L62

the lines
https://github.com/artisticat1/obsidian-latex-suite/blob/ce31511a47949e3d4d0b3a43444949fd5a6a69f6/src/editor_extensions/math_tooltip.ts#L57-L59

could be wrapped in a try catch and show an error message in the tooltip?
We could also actually deal with this and show the equation in yellow with red text like it is by default but since this not normal behaviour and we could be masking other issues with this, I'd rather not do this

**Comment by superle3** [comment]
fix is in mathjax 4.0 but it will take some time before obsidian migrates so for now try catch is used and if thus an error is caused, the text content is shown just like obsidian does.
